express-static/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+const directory = path.join('__dirname/public');
```

Comment:
This should be called with two arguments, `__dirname` and the string `'public'`.